docs/layers.md

```diff
@@ -149,9 +149,9 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8) and Natural E
 * Layer name: `buildings`
 * Geometry types: `point` and `polygon`
 
-Polygons from OpenStreetMap representing building footprint, building label_placment points, building:part features, and address points. Starts at zoom 13 by including huge buildings, progressively adding all buildings at zoom 16+. Address points are available at zoom 16+, but marked with `min_zoom: 17` to suggest that they are suitable for display at zoom level 17 and higher.
+Polygons from OpenStreetMap representing building footprint, building label_placment points, building_part features, and address points. Starts at zoom 13 by including huge buildings, progressively adding all buildings at zoom 16+. Address points are available at zoom 16+, but marked with `min_zoom: 17` to suggest that they are suitable for display at zoom level 17 and higher.
```

Comment:
Let’s fix the spelling now (it’s quick).

On Thu, Aug 4, 2016 at 10:30 AM, Matt Amos <notifications@github.com> wrote:

> In docs/layers.md
> <https://github.com/tilezen/vector-datasource/pull/944#discussion_r73566566>
> :
>
> > @@ -149,9 +149,9 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8) and Natural E
> >  * Layer name: `buildings`
> >  * Geometry types: `point` and `polygon`
> >
> > -Polygons from OpenStreetMap representing building footprint, building label_placment points, building:part features, and address points. Starts at zoom 13 by including huge buildings, progressively adding all buildings at zoom 16+. Address points are available at zoom 16+, but marked with `min_zoom: 17` to suggest that they are suitable for display at zoom level 17 and higher.
> > +Polygons from OpenStreetMap representing building footprint, building label_placment points, building_part features, and address points. Starts at zoom 13 by including huge buildings, progressively adding all buildings at zoom 16+. Address points are available at zoom 16+, but marked with `min_zoom: 17` to suggest that they are suitable for display at zoom level 17 and higher.
>
> Not a change in this PR, but I just noticed the "label_placment points"
> could be spell-fixed.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tilezen/vector-datasource/pull/944/files/24c65b9fe8cd0f939af4ef0ac239241e79a4b200#r73566566>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0EO0lNwDf0cXz3sylwUJrQkxSVAAASks5qciHHgaJpZM4Jc5j8>
> .
>